readme/image-optimization/image-optimization.md

```diff
@@ -0,0 +1,87 @@
+# 🎴 Automated assets optimization
+
+## 💫 Introduction
+
+The problem that we faced is that we experience troubles with initial page loading, because of too many images are trying to load all at once, even if this images are not yet visible on the page (especially noticeable on devices, with slow internet connection)
+
+🫢 Therefore we are facing a significant performance drop on initial page load, which is unacceptable in modern web:
+
+<img src="assets/bad-performance.png" width="600" alt="image">
+
+For faster loading, developers use modern `WebP` images and `lazy loading`. This means images only download when they appear on screen.
+
+Our motivation is that developers shouldn't think about any trivial performance optimization and focus on his more important job.
+
+Therefore we introduced an automated image optimization technique that will produce all the necessary work by himself. The optimizations are performed in a super simple way:
```

Comment:
I suggest replacing it with `work by itself`

Reply:
Done